packages/boiler-task-selenium/src/get-capabilities.js

```diff
@@ -20,10 +19,42 @@ export default function getCapabilities(config, runnerOptions = {}, forceTunnel)
   const {devPort, devUrl, internalHost} = sources;
   const {devUrl: devUrlOverride} = runnerOptions;
 
+  /**
+   * Return the base URL for all tests to be run against
+   *
+   * @return {String}
+   */
+  function getDevUrl() {
+    if (typeof devUrlOverride === 'function') {
```

Comment:
If `devUrl` in test.config.js is a function, its return value bypasses the protocol, host, and port concatenation logic below.  This is intentional for simplicity's sake!